python/ray/autoscaler/batching_node_provider.py

```diff
@@ -187,6 +199,13 @@ def _cur_num_workers(self, node_data_dict: Dict[str, Any]):
 
     def node_tags(self, node_id: str) -> Dict[str, str]:
```

Comment:
```python
def node_tags(self, node_id: str) -> Dict[str, str]:
    node_data = self.node_data_dict[node_id]
    tags = {
        TAG_RAY_NODE_KIND: node_data.kind,
        TAG_RAY_NODE_STATUS: node_data.status,
        TAG_RAY_USER_NODE_TYPE: node_data.type,
    }
    if node_data.replica_index is not None:
        tags[TAG_RAY_REPLICA_INDEX] = node_data.replica_index
    return tags
```

Reply:
Done in [a06581c](https://github.com/ray-project/ray/pull/43470/commits/a06581c75fbfae32ec5b237c4b40f2aee28af7a1).